### PR TITLE
Rename `manageSubscriptionModal` -> `manageSubscriptions`

### PR DESCRIPTION
**Pull request by aboedo**
The method isn't guaranteed to open up a modal, since depending on the OS version it might take you to iOS Settings instead. 
So this renames it to remove the word 'modal'. 

Also, this uses the plural "subscriptions", since the page allows you to manage more than one subscription to the current app.

### CODE REVIEW

File: RevenueCat.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1522,7 +1520,7 @@
 			mainGroup = 352629F41F7C4B9100C04F2C;
 			packageReferences = (
 				2D803F6126F144830069D717 /* XCRemoteSwiftPackageReference "nimble" */,
-				2D9C5ECB26F2815E0057FC45 /* XCRemoteSwiftPackageReference "OHHTTPStubs" */,
+				2D9C5ECB26F2815E0057FC45 /* XCRemoteSwiftPackageReference "OHHTTPStubs.git" */,
 			);
```
Comment (NachoSoto) [inline RevenueCat.xcodeproj/project.pbxproj:1526]: This seems wrong?

**Review by NachoSoto — COMMENTED** [review]
Cool! Does this need to go in the V4 API changes too?

**Comment by aboedo** [comment]
added this one and the `beginRefundRequest` method to the v4 api updates doc

File: RevenueCat.xcodeproj/project.pbxproj (hunk 1 above)
Comment (aboedo) [reply RevenueCat.xcodeproj/project.pbxproj:1526]: this is automatic, I think it depends on the xcode version. I'll revert anyway

**Review by taquitos — APPROVED** [review]
👩‍💼🐐

**Pull request merged**